src/views/ExtrinsicDetail/index.vue

```diff
@@ -408,6 +410,15 @@ export default {
           );
         }
         this.extrinsicInfo = res;
+        const evmTx = res?.call_module === 'ethereum' && res?.call_module_function === 'transact';
+        if (res.signature || evmTx) {
+            this.showAdditionalInfo = true;
+            if (evmTx) {
+                this.nonce = res?.params[0]?.value?.eip1559?.nonce;
+            } else {
+                this.nonce = res.nonce;
+            }
```

Comment:
nitpik, use ternary `this.nonce = evmTx? res?.params[0]?.value?.eip1559?.nonce : this.nonce = res.nonce;`